packages/@aws-cdk/aws-lambda/test/test.vpc-lambda.ts

```diff
@@ -43,6 +43,81 @@ export = {
 
       test.done();
     }
+	
+	 public 'has securitygroup that is passed in props'(test: Test) {
+	  // WHEN
+      this.lambda = new lambda.Function(this.stack, 'LambdaWithCustomSG', {
+        code: new lambda.InlineCode('foo'),
+        handler: 'index.handler',
+        runtime: lambda.Runtime.NODEJS_8_10,
+        vpc: this.vpc,
+        securityGroup: new ec2.SecurityGroup(this.stack, 'CustomSecurityGroup', { vpc: this.vpc }),
```

Comment:
Minor: suggest using a more unique phrase here than 'CustomSecurityGroup'. Could accidentally collide with a real construct encoded into the lambda module. How about 'LambdaWithCustomSG'?

Reply:
This is the SG resource-id. So I rather called it `CustomSecurityGroupX` to keep it unique now. Hope that suits.